shared/actions/chat2/index.tsx

```diff
@@ -2584,6 +2584,13 @@ const updateNotificationSettings = async (
 
 function* blockConversation(_: TypedState, action: Chat2Gen.BlockConversationPayload) {
   yield Saga.put(Chat2Gen.createNavigateToInbox({findNewConversation: true}))
+  // just reset to the people tab next time so we don't accidentally go there on start
+  yield Saga.spawn(() =>
```

Comment:
I think it'd be nice if `persistRoute` took a `clear` bool and did this. So the type stored in `ui.routeState2` is only made by that function.

Reply:
done